src/bp-templates/bp-nouveau/buddypress/groups/single/request-membership.php

```diff
@@ -6,9 +6,80 @@
  * @version 3.1.0
  */
 
-bp_nouveau_group_hook( 'before', 'request_membership_content' ); ?>
+global $bp;
+bp_nouveau_group_hook( 'before', 'request_membership_content' );
 
-<?php if ( ! bp_group_has_requested_membership() ) : ?>
+if ( groups_check_user_has_invite( bp_loggedin_user_id(), $bp->groups->current_group->id ) ) {
+
+	bp_nouveau_group_hook( 'before', 'invites_content' );
+	?>
+
+	<div class="entry-content">
+		<ul id="groups-list" class="invites item-list bp-list item-list groups-list" data-bp-list="groups_invites">
+
+			<li <?php bp_group_class( array( 'item-entry' ) ); ?> data-bp-item-id="<?php bp_group_id(); ?>" data-bp-item-component="groups">
+				<div class="list-wrap">
+
+					<?php if ( ! bp_disable_group_avatar_uploads() ) : ?>
+						<div class="item-avatar">
+							<a href="<?php bp_group_permalink(); ?>"><?php bp_group_avatar( bp_nouveau_avatar_args() ); ?></a>
+						</div>
+					<?php endif; ?>
+
+					<div class="item">
+
+						<div class="item-block">
+
+							<h2 class="list-title groups-title"><?php bp_group_link(); ?></h2>
+
+							<p class="item-meta group-details">
+								<?php $inviter = bp_groups_get_invited_by( bp_loggedin_user_id(), $bp->groups->current_group->id ); ?>
```

Comment:
@KartikSuthar can we use `bp_get_current_group_id()` instead?

Reply:
@mehulkaklotar used function instead of a global variable.